assets/__mocks__/leaflet.js

```diff
@@ -0,0 +1,166 @@
+// from https://github.com/PaulLeCam/react-leaflet/blob/master/__mocks__/leaflet.js
```

Comment:
This is the implementation of leaflet that all tests except `map.test.tsx` use. I think it would be better to have them in the tests directory, but unfortunately, that's not where they're supposed to go. https://jestjs.io/docs/en/manual-mocks